lxd/db/transaction.go

```diff
@@ -25,11 +28,20 @@ func (n *NodeTx) Tx() *sql.Tx {
 // It wraps low-level sql.Tx objects and offers a high-level API to fetch and
 // update data.
 type ClusterTx struct {
-	tx     *sql.Tx // Handle to a transaction in the cluster dqlite database.
-	nodeID int64   // Node ID of this LXD instance.
+	tx     *sql.Tx           // Handle to a transaction in the cluster dqlite database.
+	nodeID int64             // Node ID of this LXD instance.
+	stmts  map[int]*sql.Stmt // Prepared statements by code.
 }
 
 // NodeID sets the the node NodeID associated with this cluster transaction.
 func (c *ClusterTx) NodeID(id int64) {
 	c.nodeID = id
 }
+
+func (c *ClusterTx) stmt(code int) *sql.Stmt {
+	stmt, ok := c.stmts[code]
+	if !ok {
+		panic(fmt.Sprintf("no prepared statement registered with code %d", code))
```

Comment:
s/no/No/ :)

Reply:
Also, I hate panics, but I guess it's one of those cases where we don't have a choice?

Reply:
Changed from ```no``` to ```No```.

We indeed have no choice. It's basically impossible to trigger the panic in a normal release, this should be triggered only at development time if you are misusing the code generation tool.